static/scripts/coursesOverview.js

```diff
@@ -20,7 +20,7 @@ $(document).ready(function () {
                     if (member.displayName) {
                         courseMembers = courseMembers + '<li>' + member.displayName + '</li>';
                     } else {
-                        courseMembers = courseMembers + '<li>' + member.firstName + ' ' + member.lastName + '</li>';
+                        courseMembers = courseMembers + '<li>' + member.firstName + '<br>' + '&nbsp;&nbsp;&nbsp;&nbsp;' + member.lastName + '</li>';
```

Comment:
why are the &nbsp;&nbsp;&nbsp;&nbsp; needen, when we have br?

Reply:
for the spacing. after break it starts directly from the bottom of the number. adding nbsp 4 times solved the issue.